client/gutenberg/extensions/related-posts/editor.js

```diff
@@ -17,7 +17,24 @@ import { ALIGNMENT_OPTIONS, MAX_POSTS_TO_SHOW } from './constants';
 registerBlockType( 'a8c/related-posts', {
 	title: __( 'Related Posts' ),
 
-	icon: 'admin-page',
+	icon: (
+		<svg xmlns="http://www.w3.org/2000/svg">
+			<defs>
+				<path
+					id="a"
+					d="M4 5v14h17V5H4zm4 2v2H6V7h2zm-2 6v-2h2v2H6zm0 2h2v2H6v-2zm13 2h-9v-2h9v2zm0-4h-9v-2h9v2zm0-4h-9V7h9v2z"
+				/>
+			</defs>
+			<g fill="none" fillRule="evenodd">
```

Comment:
These mask fill rules are a little strange. I'm not entirely sure of support for these. It might be better to remake this icon once Mark is back. I think it's good to merge in as is for now.

Reply:
👍 Fair enough. 

I'm not an SVG wizard, so I'll ship for now, but happy to update when you folks have the improved version.